tests/test_solvermode/test_species_model.py

```diff
@@ -0,0 +1,103 @@
+def test_activate_species_transport(solver_session):
+    species_mdl = solver_session.setup.models.species.model
+    assert species_mdl.option() == "off"
+    species_mdl.option = "species-transport"
+    assert species_mdl.option() == "species-transport"
+
+
+@pytest.mark.quick
+@pytest.mark.setup
+@pytest.mark.fluent_version(">=24.1")
+def test_command_list(solver_session):
+    materials = solver_session.setup.materials
+    assert materials.mixture.command_names == [
+        "delete",
+        "list",
+        "list_properties",
+        "make_a_copy",
+    ]
+
+
+@pytest.mark.quick
+@pytest.mark.setup
+@pytest.mark.fluent_version(">=24.1")
+def test_create_custom_mixture(solver_session):
+    materials = solver_session.setup.materials
+    custom_species_1 = materials.fluid.create("custom-species-1")
+    custom_species_2 = materials.fluid.create("custom-species-2")
+    solver_session.tui.define.materials.change_create(
+        "mixture-template",  # Change/create mixture-template
+        "custom-mixture",  # Rename to `custom-mixture`
+        "yes",  # Change mixture species
+        "2",  # Set number of species to 2
+        custom_species_1.name(),  # Set species 1
+        custom_species_2.name(),  # Set species 2
+        "0",  # No surface species
+        "0",  # No site species
+        "no",  # Do not change density
+        "no",  # Do not change specific heat
+        "no",  # Do not change thermal conductivity
+        "no",  # Do not change viscosity
+        "no",  # Do not change mass diffusivity
+        "no",  # Do not change speed of sound
+        "yes",  # Overwrite mixture-template
+    )
+    assert "custom-mixture" in materials.mixture.keys()
+    assert "mixture-template" not in materials.mixture.keys()
+    mix_species = solver_session.setup.materials.mixture[
+        "custom-mixture"
+    ].species.volumetric_species.keys()
+    assert "custom-species-1" in mix_species
+    assert "custom-species-2" in mix_species
+
+
+@pytest.mark.quick
+@pytest.mark.setup
+@pytest.mark.fluent_version(">=24.1")
+def test_copy_mixture(solver_session):
+    materials = solver_session.setup.materials
+    materials.mixture.make_a_copy(
+        from_="custom-mixture",
+        to="custom-mixture-copy",
+    )
+    assert "custom-mixture-copy" in materials.mixture.keys()
+
+
+@pytest.mark.quick
+@pytest.mark.setup
+@pytest.mark.fluent_version(">=24.1")
+def test_reassign_mixture(solver_session):
+    elbow_zone = solver_session.setup.cell_zone_conditions.fluid["elbow-fluid"]
+    assert elbow_zone.material() == "custom-mixture"
+    elbow_zone.material = "custom-mixture-copy"
+    assert elbow_zone.material() == "custom-mixture-copy"
+
+
+@pytest.mark.quick
+@pytest.mark.setup
+@pytest.mark.fluent_version(">=24.1")
+def test_delete_mixture(solver_session):
+    materials = solver_session.setup.materials
+    materials.mixture.delete(name_list=["custom-mixture"])
+    assert "custom-mixture" not in materials.mixture.keys()
```

Comment:
The way these are structured now, these aren't "unit" tests, i.e. they aren't isolated. 

If I run `test_delete_mixture` on its own for example, it is always going to fail, and this is not how we've been structuring the tests. Here they only work if all the tests run, and if they run in the specific order defined in this file.

I think keeping everything as one test like we had before was ultimately better

Reply:
Yeah, agree, unit tests must work in isolation and should not have dependencies on other unit tests.